frontend/src/__tests__/components/filters/Filters.test.js

```diff
@@ -269,6 +266,7 @@ describe('Filters tests', () => {
             visible: false,
           },
         },
+        filters: filtersStoreThree,
```

Comment:
tree ? :)

Reply:
No no. It's the no of the filtersStore snapshot. Used diff snapshots and named them One, Two, Three